db/db_rate_limiter_test.cc

```diff
@@ -9,12 +9,12 @@
 
 namespace ROCKSDB_NAMESPACE {
 
-class DBRateLimiterTest
+class DBRateLimiterOnReadTest
     : public DBTestBase,
       public ::testing::WithParamInterface<std::tuple<bool, bool, bool>> {
  public:
-  DBRateLimiterTest()
-      : DBTestBase("db_rate_limiter_test", /*env_do_fsync=*/false),
+  DBRateLimiterOnReadTest()
```

Comment:
Add `explicit`.

Reply:
Thanks for the review - will change it in https://github.com/facebook/rocksdb/pull/9607 as the reviewer as requested to combining all these 3 PRs into one https://github.com/facebook/rocksdb/pull/9607#issuecomment-1056068292.